ci-operator/step-registry/ipi/install/powervs/install/ipi-install-powervs-install-commands.sh

```diff
@@ -250,8 +254,57 @@ cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
 echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_START"
 
+# Add ignition configs
+openshift-install --dir="${dir}" create ignition-configs &
+wait "$!"
+
 openshift-install --dir="${dir}" create manifests &
 wait "$!"
+# copy ccoctl files
+cat > "${dir}/manifests/openshift-cloud-controller-manager-ibm-cloud-credentials-credentials.yaml" << EOF
```

Comment:
this is fine for now..but why can't we just run the ccoctl command? permissions issues?

Reply:
Yes, there are certain limitation to the roles. In future we do plan to run ccoctl commands.